### PR TITLE
Updating Docker documentations

### PR DESCRIPTION
**Pull request by ChethanUK**

### CODE REVIEW

File: presto-docker-image/README.md [hunk 1]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
 The default configuration uses `/data/presto` as the default for
 `node.data-dir`. Thus if using the default configuration and a mounted volume
 is desired for the data directory it should be mounted to `/data/presto`.
+
+### Quickstart
+The following steps will guide to quick test PrestoSQL using docker image.
```
Comment (findepi) [inline presto-docker-image/README.md]: Please use "Presto" rather then "PrestoSQL".

File: presto-docker-image/README.md [hunk 2]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
...
+
+ 1. Run the PrestoSQL docker
+
+  To launch single machine prestosql instance for testing which will function as both a coordinator and worker, just run:
+
+  ```bash
+  docker run -p 8080:8080 prestosql/presto:latest
```
Comment (findepi) [inline presto-docker-image/README.md]: - `:latest` is default, so no need to specify it here
- give the instance a name: `--name presto`

File: presto-docker-image/README.md [hunk 3]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
...
+  ```
+  
+  NOTE: You need to have [docker](https://docs.docker.com/install/) installed in the system
+
+  Wait for the message: `INFO	main	io.prestosql.server.PrestoServer	======== SERVER STARTED ========`
+
+  Now PrestoSQL server is running at `localhost:8080` or `YOUR-INSTANCE-IP:8080`
```
Comment (findepi) [inline presto-docker-image/README.md]: I think we can omit `` or `YOUR-INSTANCE-IP:8080` `` for brevity.

File: presto-docker-image/README.md [hunk 4]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
...
+ 
+ 2. Connect to PrestoSQL 
+ 
+  To connect to PrestoSQL server: 
+  
+  Download the PrestoSQL CLI (terminal-based interactive shell for running queries) using command: 
+  ```bash
+  wget -O presto-cli-312 https://repo1.maven.org/maven2/io/prestosql/presto-cli/312/presto-cli-312-executable.jar
```
Comment (findepi) [inline presto-docker-image/README.md]: You're using hard-coded version number here, so we should be updating this per-release.... better to have something that doesn't require maintenance

i strongly feel the docker image that we recommend for demo purposes should have presto-cli preinstalled, and available on `$PATH` (perhaps in `/usr/local/bin`). I think postgresql, myql, mssql docker images all do so and I find this very convenient.

So, I suggest
1. we add the cli to the image
2. we just use `docker exec -it presto presto-cli`

File: presto-docker-image/README.md [hunk 5]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
...
+  ```
+  NOTE: [Java](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) should already be installed in system. Run `apt install openjdk-8-jre-headless` OR `brew install java`
```
Comment (findepi) [inline presto-docker-image/README.md]: When cli is prepackaged in the docker, this line becomes obsolete.

File: presto-docker-image/README.md [hunk 6]
```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
 The default configuration uses `/data/presto` as the default for
 `node.data-dir`. Thus if using the default configuration and a mounted volume
 is desired for the data directory it should be mounted to `/data/presto`.
+
+### Quickstart
```
Comment (findepi) [inline presto-docker-image/README.md]: Let's move it above `Specific Config Options`

File: presto-docker-image/README.md (hunk 4 above)
Comment (ChethanUK) [reply presto-docker-image/README.md]: Yeah!, That can help

File: presto-docker-image/README.md (hunk 1 above)
Comment (findepi) [reply presto-docker-image/README.md]: Also please update this in all the other places below.

File: presto-docker-image/README.md (hunk 4 above)
Comment (11xor6) [reply presto-docker-image/README.md]: So the docker image does contain the CLI already and it is in the path at `/usr/bin/presto`

File: presto-docker-image/README.md (hunk 5 above)
Comment (11xor6) [reply presto-docker-image/README.md]: CLI is already packaged.

File: presto-docker-image/README.md (hunk 4 above)
Comment (ChethanUK) [reply presto-docker-image/README.md]: Okay Update :)

Comment (electrum) [reply presto-docker-image/README.md]: I just tried and this works today (if you use `--name presto` when starting the container):
```
docker exec -it presto presto
```

Comment (ChethanUK) [reply presto-docker-image/README.md]: Yeah, I updated that 
https://github.com/prestosql/presto/pull/868/files#diff-67eef4356830ac58f22e2395cca9b6c6R54

File: presto-docker-image/README.md [hunk 7]
```diff
@@ -13,6 +13,54 @@ Configuration is expected to be mounted to either  to `/etc/presto` or
 `/usr/lib/presto/etc` (the latter takes precedence). If neither of these exists
 then the default single node configuration will be used.
 
+### Quickstart
+The following steps will guide to quick test Presto using docker image.
+
+ 1. Run the Presto docker
+
+  To launch single machine prestosql instance for testing which will function as both a coordinator and worker, just run:
```
Comment (findepi) [inline presto-docker-image/README.md]:
```suggestion
You can launch single-node Presto for testing purposes. The Presto node will function both
as a coordinator and a worker. To launch it, execute:
```

File: presto-docker-image/README.md [hunk 8]
```diff
@@ -13,6 +13,54 @@ Configuration is expected to be mounted to either  to `/etc/presto` or
...
+
+  OR  
+  
+  ```
+  docker exec -it presto presto --catalog tpch --schema sf1
+  ```
+  
+  Run the queries:
+  ```bash
+  presto:sf1> SELECT COUNT(*) FROM customer;
```
Comment (findepi) [inline presto-docker-image/README.md]:
```suggestion
  presto:sf1> SELECT count(*) FROM customer;
```

File: presto-docker-image/README.md [hunk 9]
```diff
@@ -13,6 +13,54 @@ Configuration is expected to be mounted to either  to `/etc/presto` or
 `/usr/lib/presto/etc` (the latter takes precedence). If neither of these exists
 then the default single node configuration will be used.
 
+### Quickstart
+The following steps will guide to quick test Presto using docker image.
```
Comment (findepi) [inline presto-docker-image/README.md]:
```suggestion
Use the following guide to quickly launch single-node Presto installation in a Docker container.
```

**Comment by OliverMorenoGonzalez** [comment]
Hello Chethan,

Since Martin tell us about coordination, I suggest work and document our work and commit one time only. One time effective commit and not multiple tiny commits.

Agree?

And coordination by slack Focus in document the image.

Ok?

**Comment by electrum** [comment]
Merged, thanks!

**Comment by electrum** [comment]
@11xor6 and I made a few minor edits and merged as f6d91f0c9242e933614b8311b5e2270ea578bcbd

This will be very helpful for people to try out Presto quickly. Thank you for your contribution.

**Pull request closed without merging**